src/ethRegistrar.ts

```diff
@@ -8,25 +8,26 @@ import {
   log
 } from '@graphprotocol/graph-ts'
 
-import {
-  createEventID, ROOT_NODE, EMPTY_ADDRESS,
-  uint256ToByteArray, byteArrayFromHex, concat
-} from './utils'
+import { byteArrayFromHex, concat, createEventID, uint256ToByteArray } from './utils'
 
 // Import event types from the registry contract ABI
 import {
   NameRegistered as NameRegisteredEvent,
   NameRenewed as NameRenewedEvent,
-  Transfer as TransferEvent,
+  Transfer as TransferEvent
 } from './types/BaseRegistrar/BaseRegistrar'
 
+import {
+  NameRegistered as ControllerNameRegisteredEventOld,
+} from './types/EthRegistrarControllerOld/EthRegistrarControllerOld'
```

Comment:
You should still listen for renew events as well; the old controller will not be deactivated immediately.

Reply:
Still listening for renew on the old controller, it's just the same event so they can both reference one.